lxd-agent/network.go

```diff
@@ -69,3 +81,139 @@ func serverTLSConfig() (*tls.Config, error) {
 
 	return tlsConfig, nil
 }
+
+// reconfigureNetworkInterfaces checks for the existence of files under ./nics in the config share.
+// Each file is named <device>.json and contains the MTU and MAC address.
+func reconfigureNetworkInterfaces() {
+	// Abort if configuration folder does not exist (nothing to do), otherwise log and return.
+	nicDir, err := ioutil.ReadDir("nics")
+	if os.IsNotExist(err) {
+		return
+	} else if err != nil {
+		logger.Error("Could not read network interface configuration directory", log15.Ctx{
+			"error": err,
+		})
```

Comment:
Pretty small nit, but given the length of the line and the context used here, I'd keep it all on one line.

Also, I think we've usually used `err` as the context key instead of `error`.

Reply:
Same goes for the other occurrences of this in this PR.